### PR TITLE
Create Deploy Task Bar class

### PR DESCRIPTION
**Pull request by jamesooden**
This is a class that understands the top bar that shows the states
deployment goes through (e.g. Satellite, RHEV, CFME, etc.).   It
provides methods to get the next and previous state and the associated page 
object.

The algorithm is complete but the information for all the different transitions is not.    Also this only adds the class.   No use of it is made of it yet (but will be in later PR's).

### CODE REVIEW

File: pages/wizard/regions/deployment_step_bar.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+    },
+}
+
+
+class DeploymentStepBar(Base):
+    """
+    This class provides navigation of the deployment
+    wizards major deployment states.   These deployment
+    states are displayed as the QCI Deployment Step Bar at
+    the top of the QCI Deoployment Wizard after the products
```
Comment (blrm) [inline pages/wizard/regions/deployment_step_bar.py]: "Deoployment"

File: pages/wizard/regions/deployment_step_bar.py [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+    def get_next_step(self):
+        return self.DSB_after_active_step
+
+    def get_prev_step(self):
+        return self.DSB_before_active_step
+
+    def get_page(self, direction='NEXT'):
+        """
+        This is the most important it method of this
```
Comment (blrm) [inline pages/wizard/regions/deployment_step_bar.py]: Should the "it" in the sentence above be removed?

**Comment by blrm** [comment]
This looks excellent. :+1:

File: pages/wizard/regions/deployment_step_bar.py (hunk 1 above)
Comment (jamesooden) [inline pages/wizard/regions/deployment_step_bar.py]: :heavy_check_mark:

File: pages/wizard/regions/deployment_step_bar.py (hunk 2 above)
Comment (jamesooden) [inline pages/wizard/regions/deployment_step_bar.py]: no.

:heavy_check_mark:

**Pull request merged**